WifiART/src/structs/UsageState.java

```diff
@@ -19,17 +19,55 @@
 public class UsageState {
     // Parameters
 
-    // The retransmissionAllowed field is true if retransmission of the LCI information is allowed.
+    /**
+     * The retransmissionAllowed field is true if retransmission of the LCI information is allowed.
+     */
     private boolean retransmissionAllowed;
-    /* The retentionExpires field is true if LCI information is allowed to expire
-       after an amount of time. */
+
+    /**
+     * The retentionExpires field is true if LCI information is allowed to expire
+     * after an amount of time.
+     */
     private boolean retentionExpires;
-    private int expireTime; // Number of hours after which retention expires.
 
-    // The staLocationPolicy field is true if additional STA location information exists.
+    /** Number of hours after which retention expires. */
+    private int expireTime;
+
+    /** The staLocationPolicy field is true if additional STA location information exists. */
     private boolean staLocationPolicy;
 
-    // TODO(dmevans) Add getter methods.
+    /**
+     * Get whether or not retransmission of the LCI information is allowed.
+     * @return The boolean value of the parameter
```

Comment:
There should always be a blank line between the initial one line description of the method and the parameters or the result.  And ditto for all methods across all classes.

Reply:
Sounds good.